sugar-stories.html

```diff
@@ -50,9 +50,26 @@ <h3>Story 1: The connection between Sugar - Students - Teachers</h3>
                 One of the first examples of end-user contributions took place in Abuja, Nigeria, site of the first OLPC pilot project. While teachers and students took to Sugar quickly, they did confront some problems. The most notable of these was that the word-processor application, Write, did not have a spelling dictionary in Igbo, the dialect used in this particular classroom (and one of the more than three-hundred languages currently spoken in Nigeria). From a conventional software-development standpoint, solving this problem (300 times) would be prohibitively expensive. But for children in Abuja, equipped with Sugar, the solution was simple: confronted with the problem of lacking a dictionary, they made their own Igbo dictionary. The did not look for others to do the work for them. The took on the responsibility themselves. The Free/Libre Software ethic built into Sugar enabled local control and innovation.</p>
                 <p class="customParagraphStyle2">
                 John Gilmore heard the about our aspiration to reach out to our end users—children—at the 2008 Libreplanet conference. He asked, “how many patches have you received from kids?” At the time, the answer to his question was zero. But over the past nine years, the situation has changed dramatically. By 2015, 50% of the patches in our new releases were coming from children (See Table); our release manager in 2015–16 (Sugar v0.108) was Sam Parkinson, a fifteen-year-old from Australia; our current release manager (Sugar v0.110) is Ignacio Rodríguez, an eighteen-year-old from Uruguay who began hanging out on our IRC channel at age ten and contributing code at age twelve.</p>
-                <div class="customMargin6">
-                    <img src="{{ site.baseurl }}/assets/data_table.png" alt="Image of Sugar commits by youth contributors" width="100%" height="200px"/>
-                </div>
+                <table class="customMargin6">
+                  <tr>
+                      <th>Release number (date)</th>
+                      <th>Total Commits</th>
+                      <th>Youth Commits</th>
+                      <th>Release note URL</th>
+                  </tr>
+                  <tr>
+                      <td>0.102 (July 2014)</td>
+                      <td>424</td>
+                      <td>108</td>
+                      <td><a href="https://wiki.sugarlabs.org/go/0.102/Notes">https://wiki.sugarlabs.org/go/0.102/Notes</a></td>
```

Comment:
I'm certainly not an HTML expert, but I tried adding <p> and </p> around the text in these areas, and it looks better, especially on a larger screen. Please try this.